test-bash.sh

```diff
@@ -73,10 +152,16 @@ tests() {
 }
 
 main() {
```

Comment:
why not have it in `bash.sh` as "test_with_default", or something?

Realistically, when copying this bash, I'd like the whole function to be "copy-paste-able"?